packages/core/integration-tests/test/cache.js

```diff
@@ -5343,7 +5343,7 @@ describe('cache', function() {
         entries: ['src/index.html'],
         async setup() {
           await overlayFS.writeFile(
-            path.join(inputDir, '.htmlnanorc'),
+            path.join(inputDir, '.htmlanorc.json'),
```

Comment:
Looks like there’s a few typos.

Reply:
Oh wow that explains the tests failing, thanks for this, totally missed that

Reply:
Easily done. 😅